shared/searchv3/dumb.js

```diff
@@ -338,19 +341,60 @@ const userInputMap: DumbComponentMap<UserInput> = {
         style: {
           width: 460,
           padding: 4,
-          boxShadow: '0 0 3px rgba(0, 0, 0, .25)',
+          borderWidth: 2,
+          borderColor: 'gray',
+          borderStyle: 'solid',
         },
       },
       userItems: chrisUsers,
       usernameText: 'Chris Hemswor',
-      showAddButton: false,
     },
   },
 }
 
+const UserInputEditable = compose(
+  withState('usernameText', 'onChangeText', ''),
+  withState('userItems', 'setUserItems', ({userItems}) => userItems),
```

Comment:
I'm a little confused by this? You pass in a prop called userItems, that get overwritten by a state-prop also called userItems?